lib/utils.coffee

```diff
@@ -88,19 +89,28 @@ module.exports =
     global.mainWindow.setBounds options
   getBounds: ->
     global.mainWindow.getBounds()
+  setScreenshotDirPath: (newPath) ->
+    if !newPath
+      if process.platform == 'darwin'
+        screenshotDir = path.join path.homedir(), 'Pictures', 'Poi'
+      else
+        screenshotDir = path.join global.EXROOT, 'screenshots'
+      fs.ensureDirSync screenshotDir
+    else
+      fs.ensureDirSync newPath, () -> setScreenshotDirPath()
+  getScreenshotDirPath: ->
+    if !screenshotDir
+      this.setScreenshotDirPath()
+    return screenshotDir
   capturePageInMainWindow: (rect, callback) ->
+    dir = this.getScreenshotDirPath()   # Be careful this method has to be called here
     global.mainWindow.capturePage rect, (image) ->
       try
         buf = image.toPng()
```

Comment:
Is there any other methods like toJpg()?
I couldn't find any documentation about this. (If anyone knows, please send me a link)